### ISSUE TITLE
Outline planning

### ISSUE THREAD

**Issue by ravl1084** [bram85/topydo#225]
Hi, I've been using topydo and I find it is much better than the original CLI. The only feature that I think is missing from topydo is the ability to use outlines to plan sequences of tasks, like the [outline](https://github.com/samuelsnyder/outline-todo.txt) add-on for todo.txt.

This is a very simple way of setting up large sequences of tasks, and especially helpful for people that repeat these sequences over and over for different projects.

**Comment by mruwek** [comment]
I believe that about 90% of mentioned addon functionality is already covered by [`dep` command](http://topydo.org/#dep). Two things are missing:

1. easy handling of recurrence (I've scrathed my head about that recently and I don't have any idea right now - any ideas @bram85?)
2. assigning multiple (sub)tasks to other (sub)tasks at once - this is actually part of #176 which was mistakenly closed and I didn't have time yet to rebase it :)

Right now your best practice should be something like (be sure to replace TODO_ID... placeholder with real value):

```
add Walk dog
add Buy a leash @shopping before:TODO_ID_OF_Walk-dog-TASK
add Adopt a dog before:TODO_ID_OF_Walk-dog-TASK
```

You should also be able to create all three tasks normally, and just assign last two of them to the first one via [column-ui](http://topydo.org/#ColumnUICommandLine):

```
dep add {} before TODO_ID_OF_Walk-dog-TASK
```

First task will then (with default configuration) disappear from the view and *only after* you complete its children (Buy a leash; Adopt a dog) it will reappear again and be ready for completion.

**Comment by mruwek** [comment]
A bit of hacky but also should work (and might be much more convenient - especially with recurring sets of tasks)



1. create file with our task-set (let's name it: **dogwalk.txt**):
```
Walk dog id:dogwalk
Buy a leash @shopping p:dogwalk
Adopt a dog p:dogwalk
```


2. `topydo add -f dogwalk.txt`
3. Let's see how the dot digraph (`topydo dep dot`) shows our tasks now:

![chart-dogwalk](https://user-images.githubusercontent.com/500710/37476900-99b03c28-2876-11e8-98ab-4b5a3ce0ae27.png)

You can even create more sequential task-sets:

1. create file with our task-set (let's name it: **death-star.txt**):
```
Create a project of a Death Star with one idiotic weak spot p:ds-build
Build a Death Star id:ds-build p:rb-spies p:rb-fleet
Let the rebel spies infiltrate the imperial services and steal the plans id:rb-spies p:ds-destroy
Wait until the Rebel Fleet arrives id:rb-fleet p:ds-destroy
Death Star destroyed - Let's build another one! id:ds-destroy
```
2. `topydo add -f death-star.txt`
3. Dot graph:

![chart-death-star](https://user-images.githubusercontent.com/500710/37477314-acc23784-2877-11e8-99ec-61c9736e04e1.png)
